k2/csrc/aux_labels.cc

```diff
@@ -90,9 +90,10 @@ namespace k2 {
 
 void AuxLabels1Mapper::GetSizes(Array2Size<int32_t> *aux_size) {
```

Comment:
void AuxLabels1Mapper::GetSizes(Array2Size<int32_t> *aux_size) **const**

A `Get` function should be `const`.

Reply:
Fixed, thanks! Note that `GetSizes` of `Connect/Determinize/RmEpsilons/...` is not `const` as we need to do most of work in it and save some varibales into the fields of class object.